src/groups/mqb/mqbi/mqbi_dispatcher.h

```diff
@@ -319,6 +319,9 @@ class Dispatcher {
   public:
     // TYPES
 
+    typedef bsl::shared_ptr<mqbi::DispatcherEvent> DispatcherEventSp;
+    typedef bslmf::MovableRef<DispatcherEventSp>   DispatcherEventRef;
```

Comment:
`DispatcherEventRef` made me think the type was `DispatcherEventSp&` above, perhaps we could use an abbreviation like `RvRef` (for "r-value reference")?